libr/search/aes_find.c

```diff
@@ -55,7 +55,7 @@ static bool aes128_key_test(const unsigned char *buf) {
 	return word1 && word2;
 }
 
-R_API int r_search_aes_update(RSearch *s, ut64 from, const ut8 *buf, int len) {
+int search_aes_update(RSearch *s, ut64 from, const ut8 *buf, int len) {
```

Comment:
use R_IPI for internal apis (functions that are meant to be used only inside the module, static for the file-specific and R_API for the public ones. Not sure if this is documented in DEVELOPERS.md but it should be